src/main/java/com/alliander/equigy/client/api/ActivationPoint.java

```diff
@@ -0,0 +1,69 @@
+/*
+SPDX-License-Identifier: MPL-2.0
+Copyright 2020 Alliander N.V.
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package com.alliander.equigy.client.api;
+
+import mjson.Json;
+
+import java.util.Objects;
+
+public class ActivationPoint {
+    private int position;
+    private int quantity;
+
+    public ActivationPoint() {
+    }
+
+    public ActivationPoint(int position, int quantity) {
+        this.position = position;
+        this.quantity = quantity;
```

Comment:
Weet je zeker dat dit een int is altijd?
Als de eenheid MWh is kan ik me voorstellen dat men ook met decimalen kan werken...

Reply:
Note: Emma heeft bevestigd dat dit een float/double is!!
_I have checked in the implementation and for the quantity is the following: TenneT requires the format to be a maximum of 5 digits before and always three digits after the period._ 
Betekent wel dat hun test data niet in overeenstemming is hiermee.

Reply:
Overigens vind ik de docu van Tennet te kort schieten. Noem vooral als je javadoc toevoegt dat dit kWh zijn...

Reply:
Ik heb de aanpassing van `int` naar `double` gedaan en de eenheid toegevoegd aan de naam van het field.